ci-operator/step-registry/openshift-pipelines/interop-tests/openshift-pipelines-interop-tests-commands.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONSOLE_URL=$(cat $SHARED_DIR/console.url)
+API_URL="https://api.${CONSOLE_URL#"https://console-openshift-console.apps."}:6443"
+export gauge_reports_dir=${ARTIFACT_DIR}
+export overwrite_reports=false
+
+# Add timeout to ignore runner connection error
+gauge config runner_connection_timeout 600000 && gauge config runner_request_timeout 300000
+
+echo "Running olm.spec to install operator..."
+CATALOG_SOURCE=redhat-operators CHANNEL=latest gauge run --log-level=debug --verbose --tags install specs/olm.spec
+
+sleep 10m
```

Comment:
Could we find a way to poll a kubernetes resource to wait until it is available and then move on rather then just add a sleep here.

Reply:
I added this sleep here just for debugging, not required to wait for any resources.